Docker/debian/debian.json

```diff
@@ -27,9 +27,87 @@
       "inline": [
         "rm -rf /etc/apt/apt.conf.d/20auto-upgrades",
         "apt-get update -y && apt-get upgrade -y",
-        "apt-get install -y systemd git vim ",
+        "apt-get install -y systemd git vim sudo openssh-server curl",
         "rm -rf /tmp/* /var/lib/apt/lists/* && apt-get clean"
```

Comment:
Long term we should put this at the end, same for other distros.  We should also probably delete the saltstack repo files at the end in the same place as this.  I don't know if you want to do that in this pr or another one later.

Reply:
On another one later, or in salt-jenkins